assets/airports/kbos.json

```diff
@@ -562,152 +529,6994 @@
+        },
+        {
+            "origin": "kbos",
+            "destination": "",
+            "category": "departure",
+            "route": "KBOS.REVSS3.REVSS",
+            "altitude": "",
+            "speed": "",
+            "method": "random",
+            "rate": 1,
+            "airlines": [
+                ["aal", 10],
+                ["aca",  3],
+                ["asa",  1],
+                ["asq",  2],
+                ["dal",  7]
+            ]
+        },
+        {
+            "origin": "kbos",
+            "destination": "",
+            "category": "departure",
+            "route": "KBOS.SSOXS4.SSOXS",
+            "altitude": "",
+            "speed": "",
+            "method": "random",
+            "rate": 7,
             "airlines": [
-                ["ual", 11],
-                ["vrd", 4],
-                ["vir", 2]
+                ["aal", 10],
+                ["amx",  3],
+                ["asq",  2],
+                ["dal",  7],
+                ["jbu", 30],
+                ["nax",  5],
+                ["nks",  5],
+                ["scx",  5],
+                ["swa", 10],
+                ["tcf",  5],
+                ["ual",  8]
             ]
         },
         {
             "origin": "",
             "destination": "kbos",
             "category": "arrival",
             "route": "PONCT.JFUND1.KBOS",
-            "altitude": [32000, 40000],
-            "speed": 320,
+            "altitude": [33000, 41000],
+            "speed": 280,
             "method": "random",
-            "rate": 10,
+            "rate": 14,
             "airlines": [
-                ["aca", 5],
-                ["asa", 3],
-                ["aal", 16],
-                ["cpa", 2],
+                ["aal",     10],
+                ["aal/long", 2],
+                ["aca",      3],
+                ["asa",      1],
+                ["asq",      2],
+                ["cpa",      1],
+                ["dal",      7],
                 ["jal/long", 2],
-                ["jbu", 35],
-                ["swa", 13],
-                ["nks", 6],
-                ["scx", 2],
-                ["ual", 11],
-                ["vrd", 4]
+                ["jbu",     40],
+                ["nks",      3],
+                ["scx",      3],
+                ["swa",     10],
+                ["tcf",      5],
+                ["ual",      8],
+                ["ual/long", 2],
+                ["vrd",      5]
             ]
         },
         {
             "origin": "",
             "destination": "kbos",
             "category": "arrival",
             "route": "AJJAY.OOSHN4.KBOS",
-            "altitude": [24000, 30000],
-            "speed": 320,
-            "method": "random",
-            "rate": 15,
-            "offset": 30,
-            "period": 90,
+            "altitude": [30000, 36000],
+            "speed": 280,
+            "method": "cyclic",
+            "rate": 7,
+            "period": 60,
+            "variation": 5,
```

Comment:
that is because `variation` is not in the list of required or accepted keys in the `spawnPatternJsonValidator`.  it is built in to the `SpawnPatternModel`, though, so that is a false negative.  

oops.